Spring-Boot/Spring-Boot-Student/src/main/java/tw/waterball/judgegirl/springboot/student/repositories/jpa/GroupData.java

```diff
@@ -27,41 +26,45 @@
     private String name;
 
     @ManyToMany(cascade = CascadeType.MERGE)
+    @JoinTable(name = "groups_students",
+            joinColumns = @JoinColumn(name = "group_id"),
+            inverseJoinColumns = @JoinColumn(name = "student_id"))
     private Set<StudentData> students = new HashSet<>();
 
+    public GroupData(Integer id, String name) {
```

Comment:
`int` instead of `Integer`

Reply:
It can change Integer to int , because when group create at first time , groupData id has been not generated , so it would throw NullPointerException.